### PR TITLE
feat: discover tab redesign

### PR DESCRIPTION
**Pull request by davidejensen**
## What does this PR change?

We have removed the `HIGHIGHTS` section and added new functionalities to `PLACES` and `EVENTS`.

### PLACES
image
- Full width section
- New filters: "Point of interest" and "Featured"
- New sorting: "Most active" and "Highest rated"
- Like/Dislike buttons on place's cards
- Views and Rating on place's cards
- UI re-design of the place's card
- UI re-design of the place's pop-up

### EVENTS
image
- Full width section
- New filters: "Featured", "Trending", "Want to go", "Frequency", "Category" and "Time" range slider
- UI re-design of the event's card
- UI re-design of the event's pop-up

## How to test the changes?

1. Launch the explorer.
2. Open the  Explore menu.
3. Go to Places section and test it.
4. Go to Events section and test it.

## Documentation of the new features:
- **Feature documentaton**: https://www.notion.so/decentraland/Discover-Tab-in-Explorer-9acd853ae105488dae88e0c509146077
- **UX design**: https://www.google.com/url?q=https://www.figma.com/file/WS38jfwHBvz3TLP3RTZ2LW/%25F0%259F%2597%25BA%25EF%25B8%258F-Discover-Tab-%257C-Explorer-%257C-06%252F23?type%3Ddesign%26node-id%3D565-10034%26mode%3Ddesign&sa=D&source=calendar&ust=1691245602002495&usg=AOvVaw18BbdZffNp4xvYYfufg4v6

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### <samp>🤖 Generated by Copilot at cb3971b</samp>

Updated the places and events section of the explore V2 feature to match the new design. Removed the highlights sub section and adjusted the layout and appearance of the places sub section. Modified the `PlacesSubSection.prefab` file and the related scripts and tests.

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/EventsSubSection/EventsSubSectionMenu/EventsSubSectionComponentController.cs [hunk 1]
```diff
@@ -235,4 +312,45 @@ public static void JumpInToEvent(EventFromAPIModel eventFromAPI)
...
+            },
+            onFail: error => { Debug.LogError($"Error receiving categories from the API: {error}"); });
+    }
+
+    private static bool IsTimeInRange(string dateTime, float lowTimeValue, float highTimeValue)
+    {
+        string startTimeString = ConvertToTimeString(lowTimeValue);
+        string endTimeString = ConvertToTimeString(highTimeValue);
+
+        TimeSpan startTime = lowTimeValue < TIME_MAX_VALUE ? TimeSpan.Parse(startTimeString) : new TimeSpan(1, 0, 0, 0);
```
Comment (Kinerius) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/EventsSubSection/EventsSubSectionMenu/EventsSubSectionComponentController.cs:344]: For readability's sake, I would avoid using `new TimeSpan` and instead use one of its static constructors like `TimeSpan.FromSeconds` or `TimeSpan.FromDays`

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentView.cs [hunk 2]
```diff
@@ -64,15 +89,105 @@ public void Start()
...
+
+        sortByDropdown.SetTitle(sortingMethodsToAdd[0].text);
+        sortByDropdown.SetOptions(sortingMethodsToAdd);
+    }
+
+    private void ClickedOnFeatured()
+    {
+        DeselectButtons();
+
+        if (filter == "only_featured=true")
```
Comment (Kinerius) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentView.cs:122]: Slap that string into a constant `ONLY_FEATURED`

**Review by Kinerius — APPROVED** [review]
LGTM

**Review by anicalbano — APPROVED** [review]
The PR has been checked on the QA side and all works as expected. Leaving the approval and review.

**Comment by anicalbano** [comment]
**Highlight** section is no longer present on the **Discover** tab:

image

**Comment by sandrade-dcl** [comment]
> **Highlight** section is no longer present on the **Discover** tab:
Yes, this is the expected behavior 👍

**Pull request merged**